src/components/WebexSettings/WebexSettings.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Tab,
+  TabContent,
+  TabList,
+  TabPane,
+  Tabs,
+} from '@momentum-ui/react';
+import {WEBEX_COMPONENTS_CLASS_PREFIX} from '../../constants';
+
+/**
+ * Webex Settings component
+ *
+ * @param {object} data Data passed to the component
+ * @param {object} data.audio Children to be rendered for the audio tab
+ * @param {object} data.video Children to be rendered for the video tab
+ *
+ * @returns {object} JSX of the component
+ */
+
+export default function WebexSettings({data}) {
+  return (
+    <div className={`${WEBEX_COMPONENTS_CLASS_PREFIX}-settings`}>
+      <Tabs tabType="tabs">
+        <TabList>
+          <Tab
+            heading="Audio"
+          />
+          <Tab
+            heading="Video"
+          />
+        </TabList>
+        <TabContent>
+          <TabPane>
+            {data.audio}
```

Comment:
the audio and video UIs won't be received via props, they will be implemented right here.

Props to add:
- className
- meetingID